lib/kontena/templates/haproxy/http_in.text.erb

```diff
@@ -23,17 +23,23 @@ listen http-in
     acl host_<%= service.name %> hdr(host) -i <%= virtual_host %>
     <% end %>
   <% end %>
-  <% if service.virtual_path %>
-    acl host_<%= service.name %>_virtual_path url_beg <%= service.virtual_path %>
+  <% if service.virtual_paths? %>
+    <% service.virtual_paths.each_with_index do |virtual_path, i| %>
+    acl host_<%= service.name %>_virtual_path<%= i %> url_beg <%= virtual_path %>
```

Comment:
Should this just be a single `acl host_foo_virtual_path` with multiple `url_beg` conditions?

It's unclear to me from the docs how this works, but based on the `acl host_*` and some other examples, I think having multiple `host_*_virtual_pathXX` ACLs shouldn't be necessary?

Reply:
Alternatively something like `acl host_<%= service.name %>_virtual_path url_beg <%= service.virtual_paths.join(' ') %>`?

Reply:
`acl host_<%= service.name %>_virtual_path url_beg <%= service.virtual_paths.join(' ')` looks lot cleaner.